general/administration/hardware-acceleration.md

```diff
@@ -87,6 +87,136 @@ services:
       # RPi 4
       - /dev/vchiq:/dev/qchiq
 ```
+## Debian Docker Nvidia Hardware acceleration:
+
+In order to achieve Hardware decode inside docker containers, some steps are needed:
+Thoses steps where wrote for Debian Stretch.
+
+Prerequisits:
+https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0)
+
+    GNU/Linux x86_64 with kernel version > 3.10
+    Docker >= 1.12
+    NVIDIA GPU with Architecture > Fermi (2.1)
+    NVIDIA drivers ~= 361.93 (untested on older versions)
```

Comment:
Just curious, have you tried these drivers? 

https://github.com/keylase/nvidia-patch

Reply:
Yes i did patched my drivers: looks to be working as, if understood the limitation of consumer GPUS, i could get 3 transcodes in parallel